frontend/src/pages/dashboard/editor/Editor.tsx

```diff
@@ -32,35 +32,18 @@ export default function Editor(props: {}) {
 				<span className={ "caption" }>{ t('dashboard.questionEditor.title.caption') }</span>
 			</div>
 			
-			<div className={ "glass" }>
-				<h2>
-					<i className={ "fi fi-rr-question" }/>
-					<span id={ "editor-question-title" }
-						  contentEditable={ hasBeenSubmitted ? false : "plaintext-only" }
-						  suppressContentEditableWarning={ true }
-						  onInput={ (e) => {
-							  let title = (e.target as HTMLSpanElement).innerText.trim();
-							  while (title.includes("  ")) title = title.replace("  ", " ");
-							  setTitle(title);
-						  } }
-						  onFocus={ (e) => {
-							  window.setTimeout(() => document.execCommand('selectAll', false, undefined), 1);
-						  } }
-						  onBlur={ (e) => {
-							  let elem = document.getElementById("editor-question-title");
-							  if (elem) elem.innerText = title;
-						  } }
-						  defaultValue={ "New Question" }
-						  style={ {
-							  borderRadius: "var(--border-radius)",
-							  outlineOffset: "var(--outline-width)"
-						  } }>
-                            { t('dashboard.questionEditor.title.placeholder') }
-                    </span>
-					?
-				</h2>
-				<p style={ { marginBottom: "calc(var(--spacing) / 2)" } }>{ t('dashboard.questionEditor.title.hint') }</p>
-			</div>
+			<input type={ "text" } id={ "editor-question-title" }
+				   minLength={ 4 } maxLength={ 50 } required
+				   onInput={ (e) => {
+					   let title = (e.target as HTMLInputElement).value.trim();
+					   while (title.includes("  ")) title = title.replace("  ", " ");
+					   setTitle(title);
+				   } }
+				   onFocus={ (e) => {
+					   window.setTimeout(() => document.execCommand('selectAll', false, undefined), 1);
```

Comment:
why is execCommand() used?
> Deprecated: This feature is no longer recommended. Though some browsers might still support it, it may have already been removed from the relevant web standards, may be in the process of being dropped, or may only be kept for compatibility purposes. Avoid using it, and update existing code if possible; see the [compatibility table](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand#browser_compatibility) at the bottom of this page to guide your decision. Be aware that this feature may cease to work at any time.

[https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand)

Reply:
Again: No critical deprecation.

Reply:
Well, we should not use any deprecreated or experimental features in our application.